code/photometry.py

```diff
@@ -0,0 +1,168 @@
+from contextlib import contextmanager
+import os
+import sys
+import warnings
+
+from astropy.stats import sigma_clipped_stats
+import numpy as np
+
+from exceptions import TractorError
+
+# temporarily let the notebook start without tractor as dependency
+try:
+    from tractor import (Tractor, PixelizedPSF, NullWCS,
+                         NullPhotoCal, ConstantSky, Image)
+
+except ImportError:
+    print("tractor is missing")
+    pass
+
+
+def calc_background(*, bkgsubimage):
+    """Measure sky noise and mean level.
+
+    Parameters:
+    -----------
+    bkgsubimage: <need type>
+        Image data from which the sky background will be calculated.
+
+    Returns:
+    --------
+    skymean: float
+        Mean of sigma-clipped background
+    skynoise: float
+        Standard deviation of sigma-clipped background
+    """
+    # suppress warnings about nans in the calculation
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", category=RuntimeWarning)
+        skymean, skymedian, skynoise = sigma_clipped_stats(
+            bkgsubimage, sigma=3.0)
```

Comment:
Nitpick: I think we can go with longer lines to avoid these line breaks, would e.g. 100 work for you?

Reply:
Yup that works. I'll change my IDE settings right now.